doc/whats_new.rst

```diff
@@ -196,6 +196,11 @@ Bug fixes
       :clas:`discriminant_analysis.LinearDiscriminantAnalysis` now returns
       correct results. By `JPFrancoia`_
 
+    - Fixed bug in :class:`linear_model.sgd_fast.SquaredEpsilonInsensitive`
```

Comment:
Also a minor nitpick, I suppose this class is not a public API and might not have a doc to be linked to. I think it might be better just to mention that in ``fixed bug in SqauredEpsilonLoss``.